frontend/packages/gitops-plugin/src/components/utils/useDefaultSecret.tsx

```diff
@@ -1,12 +1,10 @@
 import * as _ from 'lodash';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore: FIXME out-of-sync @types/react-redux version as new types cause many build errors
 import { useSelector } from 'react-redux';
 import { userStateToProps } from '@console/internal/reducers/ui';
 
 const useDefaultSecret = () => {
   const { user } = useSelector(userStateToProps);
-  const userName = _.replace(user?.metadata?.name ?? '', /[^a-zA-Z0-9-]/g, '');
+  const userName = _.replace(user?.username ?? '', /[^a-zA-Z0-9-]/g, '');
```

Comment:
Oh nice, found and fixed a bug here?

Reply:
Most likely. This was hidden by a `@ts-ignore` comment.